File: api/models/school.py
from tortoise import fields, models
from owner_type import OwnerType
from tortoise.models import Model
from teacher import Teacher

class School(Model):
    rspo = fields.IntField(pk=True)
    regon = fields.TextField()
    schoolType = fields.IntField()
    name = fields.TextField()
    address = fields.TextField()
    buildingNumber = fields.TextField()
    flatNumber = fields.TextField()
    town = fields.TextField()
    postCode = fields.TextField()
    post = fields.TextField()
    # owner = fields.IntEnumField(OwnerType)

    def __str__(self):
        return self.name

File: api/KutnoAPI/Models/School.cs
using System.Text.Json.Serialization;

namespace KutnoAPI.Models;

public class School
{
    [JsonPropertyName("rspo")]
    public int Rspo { get; set; }

    [JsonPropertyName("regon")]
    public string Regon { get; set; }

    [JsonPropertyName("schooltype")]
    public SchoolType SchoolType { get; set; }

    [JsonPropertyName("name")]
    public string Name { get; set; }

    [JsonPropertyName("address")]
    public string Address { get; set; }

    [JsonPropertyName("buildingnumber")]
    public string BuildingNumber { get; set; }

    [JsonPropertyName("flatnumber")]
    public string FlatNumber { get; set; }

    [JsonPropertyName("town")]
    public string Town { get; set; }

    [JsonPropertyName("postcode")]
    public string PostCode { get; set; }

    [JsonPropertyName("post")]
    public string Post { get; set; }

    [JsonPropertyName("owner")]
    public OwnerType OwnerType { get; set; }
    public List<CategoryValues> Categories { get; set; }

    public SchoolSummary Summary { get; set; }

}

